Pod/Classes/SEGAdjustIntegration.m

```diff
@@ -87,18 +88,34 @@ + (NSString *)extractOrderId:(NSDictionary *)dictionary withKey:(NSString *)orde
 
 - (void)identify:(SEGIdentifyPayload *)payload
 {
-    if (payload.userId != nil && [payload.userId length] != 0) {
-        [Adjust addSessionPartnerParameter:@"user_id" value:payload.userId];
+    self.segmentUserId = payload.userId;
```

Comment:
This will only save the ID to memory. When the app restarts this will be `nil` again. It should be saved to disk if you want it to persist between restarts.